client/src/fluid/FluidTypes.res

```diff
@@ -136,9 +162,11 @@ module Token = {
     | TPatternFloatPoint(ID.t, ID.t, int)
     | TPatternFloatFractional(ID.t, ID.t, string, int)
     | TPatternBlank(ID.t, ID.t, int)
+
     | TConstructorName(ID.t, string)
-    | TParenOpen(ID.t)
+    | TParenOpen(ID.t) // what paren is this? constructor paren? general?
```

Comment:
There is a feature to support expression precedence via parentheses, but it isn't enabled yet